PowerPoint Unzipped File Experiments/OneImageAndPlainTextWithLink - Readable XML Formatting.pptx/ppt/slideMasters/slideMaster1.xml

```diff
@@ -148,9 +148,10 @@
             </a:lvl1pPr>
           </a:lstStyle>
           <a:p>
-            <a:fld id="{036EEA2E-C0C0-453E-845A-3EE4D9BD6F62}" type="datetimeFigureOut">
+            <a:fld id="{CD2AE7CE-E738-4E57-84D1-A5C283E0AC8B}" type="datetimeFigureOut">
```

Comment:
lots of these changes add DIFF noise and i'm not really sure why they are needed.

Reply:
Same as above.